<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>697</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>697</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TaskManager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Task-Manager.ico</normaloff>Task-Manager.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>681</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>TaskManager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="events">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>681</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Описание</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addEvent">
       <property name="text">
        <string>Добавить задачу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearEvent">
       <property name="text">
        <string>Очистить таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveToFile">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
